website/project/model.py

```diff
@@ -3032,3 +3031,72 @@ def approve_embargo(self, user, token):
                 },
                 auth=Auth(user),
             )
+
+class DraftRegistration(AddonModelMixin, StoredObject):
+
+    is_draft_registration = True
+
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+
+    datetime_initiated = fields.DateTimeField(auto_now_add=True)
+    datetime_updated = fields.DateTimeField(auto_now=True)
+
+    branched_from = fields.ForeignField('node')
+
+    initiator = fields.ForeignField('user')
+
+    registration_metadata = fields.DictionaryField({})
+    registration_schema = fields.ForeignField('metaschema')
+    registered_node = fields.ForeignField('node')
+
+    is_pending_review = fields.BooleanField(default=False)
+
+    schema_name = fields.StringField()
+
+    storage = fields.ForeignField('osfstoragenodesettings')
+
+    # proxy fields from branched_from Node
+    def __getattr__(self, attr):
+        try:
+            return self.__dict__[attr]
+        except KeyError:
+            return getattr(self.branched_from, attr, None)
+
+    def get_comments(self):
```

Comment:
The original question is not stored with the comments and answer value

![screen shot 2015-07-10 at 10 02 19 am](https://cloud.githubusercontent.com/assets/8905795/8620031/da6a3a92-26ea-11e5-9fc2-ed059eee0099.png)